add arguments passed in from port Makefile to cmake's configure
stage (but not build stage, where they aren't accepted)

don't hide compiler command lines during build

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -200,14 +200,19 @@ class awscrt_build_ext(setuptools.command.build_ext.bu
             if osx_arch:
                 cmake_args.append(f'-DCMAKE_OSX_ARCHITECTURES={osx_arch}')
 
-        run_cmd(cmake_args)
+        cmake_config_args = cmake_args
+        if 'CMAKE_ARGS'  in os.environ:
+            cmake_config_args.append(os.environ['CMAKE_ARGS'])
 
+        run_cmd(cmake_config_args)
+
         # cmake build/install
         build_cmd = [
             cmake,
             '--build', build_dir,
             '--config', build_type,
             '--target', 'install',
+            '--verbose',
         ]
         run_cmd(build_cmd)
 
